en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/defining-endpoints-in-an-openapi-definition.md

```diff
@@ -172,6 +172,12 @@ x-wso2-endpoints:
 ```     
 ### Related Links
 
+Choreo Connect supports below endpoint features.
+
+- [High Availability/ Load Balance Endpoints]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/high-availability-for-endpoints-with-choreo-connect/) : Provides load balancing for the whole API or for a given resource.
```

Comment:
Let's call this only load balanced endpoints

Reply:
In the url we don't need to repeat the choreo connect part. Let's just. say Load balanced endpoints